tox.ini

```diff
@@ -4,48 +4,43 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py27-django{ 15, 16, 17, 18, 19, 110, 111             },
-          py35-django{             18, 19, 110, 111, 20, 21, 22 },
-          py36-django{                          111, 20, 21, 22 },
-          py37-django{                          111, 20, 21, 22 },
-          py38-django{                          111, 20, 21, 22 }
+envlist = py36-django{ 22, 30, 31, 32 },
+          py37-django{ 22, 30, 31, 32 },
+          py38-django{ 22, 30, 31, 32 },
+          py39-django{ 22, 30, 31, 32 }
+
```

Comment:
Have we tried adding django 4?

Reply:
I have now, does not work out-of-the-box, but might be worth creating a new PR for.